src/choosePokemonsPage/addChosenPokemon.ts

```diff
@@ -0,0 +1,15 @@
+import { Pokemon } from "../pokemonClass";
+import { PokemonFactory } from "../pokemonFactory";
+import { Player } from "../playerClass";
+
+export const addChosenPokemon = (
+  event: Event,
+  factory: PokemonFactory,
+  activePlayer: Player
+) => {
+  const pokemonName = (event.target as HTMLImageElement).id;
+  const newPokemonObject: Pokemon = factory.getPokemonByName(pokemonName);
+  activePlayer.addPokemon(newPokemonObject);
+  (event.target as HTMLImageElement).style.cursor = "default";
+  (event.target as HTMLImageElement).style.filter = "grayscale(100%)";
```

Comment:
why not extract "event.target as HTMLImageElement" to a variable with proper name? It is used 3 times!